### DISCUSSION TITLE
Step by Step guide please

### DISCUSSION THREAD

**Discussion by ayskura** [marcel-licence/esp32_midi_sampler#99] [General]
is it possible to have a step by step guide on how to make it work? I bought an ESP32 AudioKit 2.2 but have some difficulties to flash it and see where to plug what

**Comment by marcel-licence** [comment]
In case you are able to flash the firmware you will get some output on the serial monitor. I tried to add information of the used pins on the audio kit. Unfortunately there are different versions with small changes on the pin layout making it a bit hard to detect the correct configuration.

I could try to prepare a video soon when I will find a bit time.

**Reply by ayskura** [reply]
I bought exactly the Audiokit 2.2 as the one you use in the video for this purpose Would be good to see at least how to make it happen, as I am quite new to ESP32 world

**Reply by marcel-licence** [reply]
I made some tutorials you can find on my channel. If you need further information please let me know. I could make some new ones =)
